examples/misc/lib/library_tour/core/collections.dart

```diff
@@ -1,6 +1,6 @@
 void miscDeclAnalyzedButNotTested() {
   var fruits = List<String>();
-  // ignore_for_file: stable, dev, argument_type_not_assignable
+  // ignore_for_file: stable, dev, argument_type_not_assignable, unused_local_variable
```

Comment:
Should we add beta to this list? Or remove the release names?

(Is "beta" even supported? I don't see examples/strong/analyzer-results-beta.txt in this PR.)

Reply:
Not sure? I can't quite figure out how that works. If there isn't a `-beta.txt` file it uses `analyzer-results.txt`, and I think most of the beta files that existed were fine.